userspace/sysdig/csysdig.cpp

```diff
@@ -916,6 +905,18 @@ sysdig_init_res csysdig_init(int argc, char **argv)
 #if defined(HAS_CAPTURE) && !defined(_WIN32)
 				else if(bpf)
 				{
+					if (bpf_probe.empty())
```

Comment:
When in bpf mode AND bpf_probe is empty, we overwrite bpf_probe with the default path.